dev/files/authn-jenkins/myapp.yml

```diff
@@ -0,0 +1,35 @@
+- !policy
+  id: myapp
+  body:
+  - &variables
+    - !variable database/username
+    - !variable database/password
+
+  # Create a group that will have permission to retrieve variables
+  - !group secrets-users
+
+  # Give the `secrets-users` group permission to retrieve variables
+  - !permit
+    role: !group secrets-users
+    privilege: [ read, execute ]
+    resource: *variables
+
+  # Create a layer to hold this application's hosts
+  - !layer
+
+  # Create a host using the namespace `aws` to identify this as an AWS resource.
```

Comment:
Looks like this comment might be referencing the `Authn-IAM` sample policy.

Reply:
Resolved in this commit: https://github.com/AndrewCopeland/conjur/commit/b69b6de44c479288cb5515f8493775bef157c1db